mllib/src/main/scala/org/apache/spark/mllib/tree/DecisionTree.scala

```diff
@@ -737,7 +737,12 @@ object DecisionTree extends Serializable with Logging {
     val leftWeight = leftCount / totalCount.toDouble
     val rightWeight = rightCount / totalCount.toDouble
 
-    val gain = impurity - leftWeight * leftImpurity - rightWeight * rightImpurity
+    val gainRatio = impurity - leftWeight * leftImpurity - rightWeight * rightImpurity
+  //calculate splitInfo
```

Comment:
All of these lines don't match the project code style. Please read https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark